pymatgen/io/openmm/generators.py

```diff
@@ -226,17 +226,13 @@ def get_input_set(
         openff_topology = get_openff_topology(openff_counts)
         openmm_topology = openff_topology.to_openmm()
 
+        assert np.all(ffs == ffs[0]), "All Molecules must use the same force field"
```

Comment:
This is what I had previously implemented but this isn't generally true. 

1. We might want to parameterize water differently than everything else.
2. We might need to include a few custom parameters from another force field.

You don't need to implement either, but it'd be good to stay flexible so we can add these later on.